.golangci.yml

```diff
@@ -21,7 +21,6 @@ linters:
     - govet
     - ineffassign
     - interfacer
-    - maligned
```

Comment:
Why was this removed?

Reply:
maligned was deprecated (see https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1259)